### ISSUE TITLE
컴퓨터과학 하위 목록 생성이 필요합니다

### ISSUE THREAD

**Issue by pikpokjeon** [DawitJung/visual-study#10]
1.메인 페이지에서 컴퓨터 과학을 누르면 컴퓨터 과학에 대한 개요가 나오고 여기서 알고리즘 부분으로 이동
2.메인 페이지에서 컴퓨터과학 > 알고리즘 과 같이 하위 목록을 보여줘서 메인에서 바로 해당부분 이동

을 생각하고 있는데 어떤 방식이 유저 친화적일까요?

**Comment by pikpokjeon** [comment]
실수로 태깅을 못했네요.

**Comment by DawitJung** [comment]
컴퓨터 과학 하위에 알고리즘을 만드는 것은 물론 좋습니다.

지금은 메인페이지에서 두번째 위계까지 노출하고 있습니다.
따라서 메인 페이지에서 컴퓨터 과학으로 바로 이동하는 것이 가능합니다.
/computer/computer-science에 컴퓨터 과학의 개요를 쓸 수 있는 문서가 이미 마련되어 있으며 내용을 채울 준비를 하고 있습니다.

저는 그저 단순하기때문에 1번 방법을 채택하고 있는데 다른 이유가 있다면 2번 방법도 충분히 고려할만 하다고 생각합니다.

**Comment by pikpokjeon** [comment]
네 2번째 뎁스까지 노출이 가능하니 컴퓨터과학 > 알고리즘으로 이동하여, 알고리즘 개요문서 상단 목차에 정렬 부분을 추가해서 작업하고 있습니다.
